package/MDAnalysis/auxiliary/EDR.py

```diff
@@ -0,0 +1,168 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+EDR auxiliary reader --- :mod:`MDAnalysis.auxiliary.EDR`
+========================================================
+
+EDR files are binary files following the XDR protocol. They are written by
+GROMACS during simulations and contain  the time-series energy data of the
+system.
+
+panedr is a Python package ( https://github.com/mdanalysis/panedr ) that reads
+these binary files and returns them human-readable form, either as a Pandas
+DataFrame or as a dictionary of NumPy arrays. It is used by the EDR auxiliary
+reader to parse EDR files. As such, a dictionary with string keys and numpy
+array values is loaded into the EDRReader.
+
+The EDR auxiliary reader takes the output from panedr and loads the energy data
+as auxiliary data into Universes. Standalone usage is also possible, where the
+energy terms are extracted without associating them with the trajectory, for
+example, to allow easy plotting of the energy terms.
+
+
```

Comment:
This is perhaps a down the track thing, but it would be good to give a self contained example here.

Reply:
Yeah good call, I've added this as a to-do here